src/osd/OSDMap.cc

```diff
@@ -470,6 +470,9 @@ void OSDMap::Incremental::encode(bufferlist& bl, uint64_t features) const
     if (target_v >= 4) {
       ::encode(new_require_min_compat_client, bl);
     }
+    if (target_v >= 5) {
+      ::encode(new_require_osd_release, bl);
```

Comment:
I was originally going to do that, but (1) we don't have the OSDMap so we can't initialize new_flags if it is still -1, and (2) the only daemons that care about these flags or require_osd_release are the mon and osd, and once require_luminous is set they will *always* have them set.  I think the only exception would be if someone tries to start an old daemon; the mon will be locked out due to the existing required mon feature, and the osd will be prevented from booting by the mon.